### PR TITLE
showing if superlike or not

### PR DESCRIPTION
**Pull request by mayeaux**

### CODE REVIEW

**Comment by mayeaux** [comment]
No merge?

**Comment by stuwil** [comment]
Whoops, closed by accident.

**Comment by mayeaux** [comment]
Np, let me fix merge conflicts and push it again. Is there a better way to fix the merge conflicts then just a normal merge commit, maybe I should use the rebase flag?

File: desktop-app/index.html [hunk 1]
```diff
@@ -79,7 +79,6 @@
           Superlikes Remaining: {{superLikesRemaining}}
         </span>
         <br>
```
Comment (stuwil) [inline desktop-app/index.html]: Can we drop this as well?

**Comment by stuwil** [comment]
You can do a normal merge commit. Once it can be automatically merged via GitHub, use the squash option to reduce it to a single commit.

File: desktop-app/index.html [hunk 2]
```diff
@@ -158,7 +157,8 @@ <h3 class="fader" style="display: inline;">{{allPeople[peopleIndex].distance_mi}
          class="match">
       <img ng-src="{{match.thumbnail || 'img/anonymous.png'}}" class="thumbnail"/>
       <div>
-        <p class="name">{{match.name || 'Unknown'}}</p>
+        <p class="name">{{match.name || 'Unknown'}} 
+        <i ng-show="match.isSuperLike" class="fa fa-star"></i> </p>
```
Comment (stuwil) [inline desktop-app/index.html]: Just needs indentation fixed for this block

File: desktop-app/index.html (hunk 1 above)
Comment (mayeaux) [inline desktop-app/index.html]: Sorry, you want to remove 'Superlikes Remaining' ?

Comment (stuwil) [inline desktop-app/index.html]: No, the `<br>` tag please.

Comment (mayeaux) [inline desktop-app/index.html]: Yup should be no problem.

File: desktop-app/index.html (hunk 2 above)
Comment (mayeaux) [inline desktop-app/index.html]: Yup, agreed, will fix

**Comment by mayeaux** [comment]
Will get this done this weekend @stuwil

**Pull request merged**